bindings/common-lisp/keystone.lisp

```diff
@@ -56,6 +56,10 @@
 (defmethod initialize-instance :after ((engine keystone-engine) &key)
   (with-slots (architecture mode handle) engine
     (setf handle (foreign-alloc 'ks-engine))
+    (when (consp mode)
```

Comment:
You should change the type of mode so that it can be either a keyword or a list, currently it is set just to keyword.  Should, we enforce that mode is always a list?